rules/RESPONSE-980-CORRELATION.conf

```diff
@@ -50,18 +50,30 @@ SecRule &TX:'/AVAILABILITY\\\/APP_NOT_AVAIL/' "@ge 1" \
     skipAfter:END-CORRELATION"
     SecRule &TX:'/WEB_ATTACK/' "@ge 1" "t:none"
 
-SecRule TX:INBOUND_ANOMALY_SCORE "@gt 0" \
+# Creating a total sum of all triggered inbound rules, including the ones only being monitored
+SecAction \
+    "id:980115,\
+    phase:5,\
+    pass,\
+    t:none,\
+    nolog,\
+    noauditlog,\
+    setvar:'tx.monitor_anomaly_score=%{tx.anomaly_score_pl1}',\
+    setvar:'tx.monitor_anomaly_score=+%{tx.anomaly_score_pl2}',\
+    setvar:'tx.monitor_anomaly_score=+%{tx.anomaly_score_pl3}',\
+    setvar:'tx.monitor_anomaly_score=+%{tx.anomaly_score_pl4}'"
+    
```

Comment:
This line contains spaces and makes travis to fail

Reply:
fixed